lib/Inst/Inst.cpp

```diff
@@ -876,6 +974,27 @@ Inst *souper::getInstCopy(Inst *I, InstContext &IC,
   return Copy;
 }
 
+Inst *souper::instJoin(Inst *I, Inst *EmptyInst, Inst *NewInst,
+                       std::map<Inst *, Inst *> &InstCache,
+                       InstContext &IC) {
+  std::vector<Inst *> Ops;
+
+  for (auto const &Op : I->Ops) {
+    auto NewOp = instJoin(Op, EmptyInst, NewInst, InstCache, IC);
+    Ops.push_back(NewOp);
+  }
+
+  Inst *Copy = 0;
+  if (I == EmptyInst) {
+    Copy = NewInst;
+  } else if (I->K == Inst::Var) {
+    Copy = I;
+  } else
+    Copy = IC.getInst(I->K, I->Width, Ops);
```

Comment:
if the "if" part has curly braces, then use curly braces for all subsequent parts of the conditional